python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -179,7 +214,7 @@ def setUp(self, gangadir=None, wipe_repo=None, extra_opts=[]):
         # This is called before each unittest
         if gangadir is None:
             import os
-            gangadir = os.path.join('$HOME/gangadir_testing')  # TODO make separate dirs per test suite, `self.__class__.__name__`
+            gangadir = os.path.join('$HOME/gangadir_testing', self.__class__.__name__)
```

Comment:
I just ran a test and I noticed that there was some bleed of metadata knowledge. Fixing in a small addendum to the PR now :D